src/web/js/trove/plot-lib.js

```diff
@@ -910,14 +910,14 @@
 
 
     var dimension = getDimension({
-      minWindowWidth: 600,
-      minWindowHeight: 400,
-      outerMarginLeft: 160,
-      outerMarginRight: 160,
-      marginLeft: 0,
-      marginRight: 0,
-      marginTop: 50,
-      marginBottom: 10,
+      minWindowWidth: 700,
+      minWindowHeight: 550,
+      outerMarginLeft: 10,
+      outerMarginRight: 10,
+      marginLeft: 120,
+      marginRight: 120,
+      marginTop: 90,
+      marginBottom: 40,
```

Comment:
I move labels outside of the chart to avoid collision, but that means the chart will take more space